auto-ipsec/libreswan/README.md

```diff
@@ -1,36 +1,36 @@
-* once the bootstrap key has been derived, keylime will decrypt and extract the zipped payload that includes both the cert/key and the ipsec config.
+* The tenant will first run generate.py in this folder to generate the IPsec configuration files to pass to the tenant. The tenant must specify the subnets/ips that they wish for agents to always use IPsec to communicate.
+* The tenant will generate a new CA (if one hasn't been generated yet) and then generate a new cert/key combo for the agent to be bootstrapped
+* The tenant will include the IPsec config files generated above with the `--include` option
+* Tenant and verifier will bootstrap the agent and derive a key as normal
+* Once the bootstrap key has been derived, keylime will decrypt and extract the zipped payload that includes both the cert/key and the ipsec config.
 * Then keylime will run the provided autorun.sh script.  This script will install/configure IPsec using the certificate provided.
-* at this point, IPsec will initialize and run between any specified IPs or subnets.
+* At this point, IPsec will initialize and run between any specified IPs or subnets.
 
 To support revocation:
 
-* the tenant must host the CA's CRL and listen for revocations.  `keylime_ca -c listen` will do this job
+* The tenant must host the CA's CRL and listen for revocations.  `keylime_ca -c listen` will do this job
 * If the verifier determines that an agent has failed its integrity check, it will create and sign a revocation notice that it will distribute to all the agents over 0mq
-* the tenant's CRL listener will receive and validate the revocation, update the CRL using the CA private key, and publish it to a locally running websever.
-* in parallel, all the other agents in the network will also receive the revocation notice and retrieve the new CRL from the tenant.  It is important that the certificates include the appropriate address for the CRL or this won't work.
-* the other agents in the network will also search their active IPsec security associations for the revoked agent and force IKE re-negotiation with them.  Because the CRL has also been updated, this will cause the agents to be unable to do key agreement with the revoked agent and block any traffic to it.
+* The tenant's CRL listener will receive and validate the revocation, update the CRL using the CA private key, and publish it to a locally running websever.
+* In parallel, all the other agents in the network will also receive the revocation notice and retrieve the new CRL from the tenant.  It is important that the certificates include the appropriate address for the CRL or this won't work.
+* The other machines in the network will also search their active IPsec security associations for the revoked agent and force IKE re-negotiation with them.  Because the CRL has also been updated, this will cause the agents to be unable to do key agreement with the revoked agent and block any traffic to it.
```

Comment:
This was updated in racoon but not libreswan: 

Should be "this will cause the **machines** to be unable to do key agreement with the **machine running** the revoked agent and block any traffic to it." to match racoon